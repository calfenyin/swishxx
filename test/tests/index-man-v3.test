index | | -e man:*.[1-9] -r -v3 | . | 0
