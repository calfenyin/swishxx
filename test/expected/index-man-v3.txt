
.:
  ad.1 (856 words)
  doc.4 (181 words)
  html2pdbtxt.1 (300 words)
  pdb.4 (187 words)
  pdbtxt2html.1 (199 words)
  txt2pdbdoc.1 (309 words)
  wrap.1 (444 words)
  wrapc.1 (332 words)
  wraprc.5 (206 words)

index: ranking index...
  "author" discarded (100%)
  "copy" discarded (100%)
  "copyright" discarded (100%)
  "description" discarded (100%)
  "details" discarded (100%)
  "distributed" discarded (100%)
  "file" discarded (100%)
  "fitness" discarded (100%)
  "foundation" discarded (100%)
  "free" discarded (100%)
  "general" discarded (100%)
  "gnu" discarded (100%)
  "hope" discarded (100%)
  "implied" discarded (100%)
  "license" discarded (100%)
  "lucas" discarded (100%)
  "merchantability" discarded (100%)
  "modify" discarded (100%)
  "name" discarded (100%)
  "option" discarded (100%)
  "particular" discarded (100%)
  "paul" discarded (100%)
  "program" discarded (100%)
  "public" discarded (100%)
  "published" discarded (100%)
  "purpose" discarded (100%)
  "received" discarded (100%)
  "redistribute" discarded (100%)
  "software" discarded (100%)
  "terms" discarded (100%)
  "useful" discarded (100%)
  "version" discarded (100%)
  "warranty" discarded (100%)
index: writing index...

index: done:
  9 indexed
  7629 words, 3014 indexed, 755 unique

